roles/openshift_prometheus/templates/prometheus.rules.j2

```diff
@@ -0,0 +1,13 @@
+groups:
+- name: example-rules
+  interval: 30s # defaults to global interval
+  rules:
+  - alert: Node Down
+    expr: up{job="kubernetes-nodes"} == 0
+    annotations:
+      miqTarget: "ContainerNode"
+      severity: "HIGH"
+      message: "{{ '{{' }}{{ '$labels.instance' }}{{ '}}' }} is down"
+{% if openshift_prometheus_external_rules is defined and openshift_prometheus_external_rules is not none %}
+  - {{ openshift_prometheus_external_rules }}
+{% endif %}
```

Comment:
@zgalor we probably want to remove the `example-rules` for now (wouldn't it be deployed by default now?).
Let's wait to add official rules when we'll have them.

Also, the `openshift_prometheus_external_rules` could it be a file that is just copied in an additional `ConfigMap`?